### ISSUE TITLE
"Janthir Wilds" Content update

### ISSUE THREAD

**Issue by queicherius** [gw2efficiency/issues#1812]
> ## This is a work-in-progress. There will be problems.

- [X] Update items job to handle new item categories
- [X] Add homestead recipes to crafting calculator  `<Blocked by API>`
  - [X] Blocked: [https://github.com/gw2-api/issues/issues/96](https://github.com/gw2-api/issues/issues/96), so #1858
  - [X] Add vendor recipes for homestead materials (with no upgrades, show a warning about potential efficiency loss and weekly gating)
  - [X] Add handling for new crafting profession (filters etc, requires recipes to be there first)
- [X] Update mastery filters for achievements & unlock statistics
  - [X] Follow-up: [https://github.com/gw2-api/issues/issues/97](https://github.com/gw2-api/issues/issues/97)
- [X] Update story quests
- [X] Wallet currencies
  - [X] Update recipes that used Unusual Coins to Ancient Coins
- [X] Update gemstore items
- [X] Add new page for homestead decorations
  - [X] Add `/v2/homestead/decorations`, `/v2/account/homestead/decorations` and `/v2/homestead/decorations/categories` to API client
  - [X] [64d2fe3f-4a3a-4193-b468-93d02ce7f8e1](https://linear.app/devoxa/issue/64d2fe3f-4a3a-4193-b468-93d02ce7f8e1)
- [X] Show equipped homestead glyphs somewhere (on decorations page?)
  - [X] Add `/v2/homestead/glyphs` and `/v2/account/homestead/glyphs` to API client
  - [X] [ccb78fb8-0944-41ab-a32f-1735994781e0](https://linear.app/devoxa/issue/ccb78fb8-0944-41ab-a32f-1735994781e0)
- [X] #1832
- [X] Add new maps to map completion
- [X] Add new hero's choice chest to daily page (and update anything that's missing in our data from `/v2/mapchests`, some of those are currently manual)
- [X] Add new content to statistics:
  - [X] Hero's choice chest
  - [X] Ursus Oblige

**Comment by noonfyi** [comment]
To go with Wallet Currencies:
All existing usage of Unusual Coins will need to be renamed to [Ancient Coin](https://wiki.guildwars2.com/wiki/Ancient_Coin) with a new icon, and a new Unusual Coin currency will need to be added.

**Comment by noonfyi** [comment]
To go with Wallet Currencies, as with EoD -> SotO launch, the SotO recipes that previously used Unusual Coins now use [Ancient Coins](https://wiki.guildwars2.com/wiki/Ancient_Coin) and their recipes will need to be switched.

**Comment by darthmaim** [comment]
> Add new hero's choice chest to daily page (and update anything that's missing in our data from /v2/mapchests, some of those are currently manual)

Those have actually all been added, see gw2-api/issues#37

**Comment by queicherius** [comment]
> Those have actually all been added, see [gw2-api/issues#37](https://github.com/gw2-api/issues/issues/37)

Ye, that's what that point is for. Thanks for updating the issue, that's how I found out 😄

**Comment by ILC-IT** [comment]
Hi I think tab number 18 of bank does not display accurate data. Only displays 2/30 materials there.

**Comment by darthmaim** [comment]
> Hi I think tab number 18 of bank does not display accurate data. Only displays 2/30 materials there.

Yes. This is a current issue with the GW2 API, not gw2efficiency though. See also https://github.com/gw2-api/issues/issues/95.

**Comment by queicherius** [comment]
Moved remaining tasks to their own issues.
